bootstrap/004_labels.sql

```diff
@@ -1,9 +1,10 @@
 
--- we should remove this and the other creaste statement in favour of driving the creation of htese tables from the python model
 CREATE TABLE INTERNAL.LABELS if not exists (name string, group_name string null, group_rank number, label_created_at timestamp, condition string, enabled boolean, label_modified_at timestamp, is_dynamic boolean, label_id string);
 
 CREATE TABLE INTERNAL.PREDEFINED_LABELS if not exists (name string, group_name string null, group_rank number, label_created_at timestamp, condition string, enabled boolean, label_modified_at timestamp, is_dynamic boolean);
 
+create table if not exists internal.validate_labels(sql text, message text, simple boolean);
```

Comment:
Should be `create or replace table`? After each upgrade, we would re-insert the same rows.